tests/catch/hipTestMain/config/config_amd_linux.json

```diff
@@ -1,7 +1,9 @@
 {
     "DisabledTests":
 	[
-	   "Unit_hipStreamPerThread_DeviceReset_1"
+	   "Unit_hipStreamPerThread_DeviceReset_1",
+	   "Unit_hipMemGetInfo_Negative",
+	   "# The prior tests relates to ticket EXSWCPHIPT-61"
```

Comment:
Write that info inside the test as a comment or something

Reply:
Changes to the json files have been removed, added ```HIP_SKIP_TEST``` inside test cases with relevant information